bulk_grades/api.py

```diff
@@ -61,10 +60,11 @@ def _get_enrollments(course_id, track=None, cohort=None):
             'enrolled': enrollment.is_active,
             'track': enrollment.mode,
         }
-        try:
-            pce = enrollment.programcourseenrollment.program_enrollment
-            enrollment_dict['student_uid'] = pce.external_user_key
-        except ObjectDoesNotExist:
+        program_course_enrollment = enrollment.programcourseenrollment_set.all()
+        if program_course_enrollment.exists():
+            program_course_enrollment = program_course_enrollment.first().program_enrollment
+            enrollment_dict['student_uid'] = program_course_enrollment.external_user_key
+        else:
```

Comment:
Will the `program_enrollment` object be the same for all the PCE? Just confirming this to avoid any incomplete/incorrect data export.